src/main/kotlin/com/github/warningimhack3r/intellijshadcnplugin/backend/sources/Source.kt

```diff
@@ -73,7 +75,8 @@ abstract class Source<C : Config>(val project: Project, private val serializer:
         } ?: FileManager.getInstance(project).getFileContentsAtPath(configFile)?.let {
             log.debug("Parsing config from $configFile")
             try {
-                Json.decodeFromString(serializer, it).also {
+                val json = Json { ignoreUnknownKeys = true }
```

Comment:
My guess is that this is the only place where this modification is useful, all the others are not related to the `Config` class and are only used to get `JsonObject` instances, they are not restrictive